### ISSUE TITLE
Add no-break spaces in Morse shorthand?

### ISSUE THREAD

**Issue by tuskpot** [standardebooks/ella-cheever-thayer_wired-love#4]
The book frequently uses station codes and Morse code shorthand. These are often two letters, separated by a space. Occasionally my reader split them across a line break, which seems undesirable. Would you accept a pull request to replace spaces with no-break spaces in these shorthand usages?

Here are some of the shorthand that I would suggest putting a no-break space in:

- "B m"
- "X n"
- "B a"
- "G. A."
- "O. K."

If you're interested, I'll do a closer review to see if there are any others, or any exceptions, before submitting a pull request.

Here are some examples in context:

> […] Nattie was surprised to hear the old familiar “B m⁠—B m⁠—B m⁠—X n.”

> With despair in her heart, once more Nattie interrupted “X n,” took the impatient gentleman’s message, studied out its illegible characters, […]

> “Do you think it will help the matter at all for you to make a display of your charming disposition? \<abbr\>G. A.\<\/abbr\>⁠—the⁠—.”

> But after she had given “\<abbr\>O. K.\<\/abbr\>” and closed her key, he opened his[…]

> “C said this morning he was going to B a⁠—and⁠—”

**Comment by acabal** [comment]
Yes, I think that's a good idea. A complete review would be ideal before you begin. Thanks!

**Comment by tuskpot** [comment]
Here's a complete inventory of these codes and abbreviations. These were identified by using the regex `\b[a-z]{1}[\.]? [a-z]{1}[\.]?\b` in case-insensitive mode, followed by some additional review.

Let me know if you have any feedback.

**Station codes**
- `B m` — The station where Nattie works. (appears 28 times in 7 chapters, plus an end-note)
- `X n` — The station where Clem works. (appears 38 times in 8 chapters, plus an end-note)
- `B a` — Another station, which Clem visits at one point. (appears 4 times in 2 chapters)
- `P m` — A clear printing error that should be corrected to `B m`. (appears once)

**Abbreviations**
- `G. A.` — An telegrapher's abbreviation for "go ahead". (appears 6 times in 1 chapter)
- `g a` — A different styling for "go ahead". I would standardize this to `G. A.` instead. (appears once, in a different chapter than `G. A.`)
- `O. K.` — A formal closing used to end business communications over the telegraph. (appears 10 times in 5 chapters)
  - Ch. 1: Every operator is obliged to sign his own private “call,” as well as the office “call,” and “\<abbr\>O. K.\<\/abbr\>” at the close of each message.
- `G. M.` — An abbreviation for "good morning". (appears 3 times in 2 chapters)
- `g. n.` — An abbreviation for "good night". I would standardize this to `G. N.` to match the uppercase used in all other abbreviations. I don't see any reason for it to be different. (appears 3 times in 1 chapter)

**Other**
- `H a` — The letters "H" and "a", spelling out "Ha". (appears once)
  - Ch. 1: To her surprise and displeasure “X n” laughed⁠—the circumstance being conveyed to her understanding in the usual way, by the two letters “H a!”
- `M. R. F.` — A reference to character in a Dickens book who refers to "my respected father" using the "M. R. F." abbreviation. It's not a name, so I don't think name semantics apply, but it is an abbreviation and should be tagged with \<abbr\>\</abbr\>. (appears once)
  - From Wired Love, Ch. 10: […]my father, with a pigheadedness worthy of Eugene Wrayburn’s M. R. F. in ‘Our Mutual Friend,’ determined to make a doctor of me[…]
  - From Dickens' [Our Mutual Friend](https://www.gutenberg.org/files/883/883-h/883-h.htm), Ch. 12: "My respected father--let me shorten the dutiful tautology by substituting in future M. R. F., which sounds military, and rather like the Duke of Wellington."

So, to recap, I'm proposing the following changes, probably split across four commits:

1. Correct `P m` to `B m`.
2. Standardize abbreviations:
2a. Standardize `g a` to `G. A.` and tag it with \<abbr\>\</abbr\>.
2b. Standardize `g. n.` to `G. N.`
3. Add \<abbr\>\</abbr\> tags to `M. R. F.`.
4. In all the items listed in the sections above, replace spaces with non-breaking spaces.

**Comment by tuskpot** [comment]
Oh, and I just realized that "Our Mutual Friend ought to be italicized and semanticated as a novel, so I would make that change as well.

**Comment by acabal** [comment]
OK, that all sounds good. Note that [OK is not an abbreviation](https://standardebooks.org/manual/1.6.1/single-page#8.10.9.1) though. You can leave it with periods and spaces to match the rest of the style. Thanks!

**Comment by tuskpot** [comment]
Yeah, I wondered about that. The abbr tags are already in there, but I can remove them around the "O. K." codes as a separate commit.
